src/site/_data/i18n/tags.yml

```diff
@@ -298,6 +298,24 @@ cors:
     ru: Последние новости и статьи о CORS.
     zh: 我们关于 CORS 的最新资讯、动态和故事。
 
+cross-browser:
+  title:
+    en: Cross-browser
```

Comment:
I don't want to call it cross browser as per my comment below. It's more "newly interoperable" which is not a very snappy tag to be fair. I'll have a think.

Reply:
Your suggestion `newly-interoperable` has the small problem that it ages poorly. What's new today is old tomorrow. Other ideas:

`universally-usable`?
`universally-supported`?

Reply:
It aging badly is the entire point. As mentioned, I'd intend to remove the tags because everything on the site is intended to be interoperable so there would be no point to the tag if we added it to everything ;) The aim is to show case the stuff that's landed within a recent period.

Reply:
In this case it makes sense. I just brought it up because whenever we would write "new" in a post, Joe would make us remove it. This is different, though, so `newly-interoperable` works for me then. Changing to that.